persistence-dse-6.8/pom.xml

```diff
@@ -211,6 +221,23 @@
               </artifactItems>
             </configuration>
           </execution>
+          <execution>
+            <id>unpack-dnsjava</id>
+            <phase>compile</phase>
+            <goals>
+              <goal>unpack</goal>
```

Comment:
We need the same hack as sjk-core, because dnsjava also declares types in the root package.